mscore/synthcontrol.ui

```diff
@@ -129,9 +129,9 @@
     </layout>
    </item>
    <item row="0" column="1">
-    <widget class="Awl::VolSlider" name="mgain">
+    <widget class="Awl::VolSlider" name="mgain" native="true">
      <property name="sizePolicy">
-      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
+      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
```

Comment:
This is the essential change.  All the other code changes here must have been things QtCreator did automatically.